auth/src/main/java/com/firebase/ui/auth/viewmodel/idp/SocialProviderResponseHandler.java

```diff
@@ -78,12 +81,28 @@ public void onSuccess(AuthResult result) {
                 .addOnFailureListener(new OnFailureListener() {
                     @Override
                     public void onFailure(@NonNull Exception e) {
-                        if (e instanceof FirebaseAuthUserCollisionException) {
+                        // For some reason disabled users can hit FirebaseAuthUserCollisionException
```

Comment:
This feels like a Firebase Auth bug, I don't think that error class and that error exception should ever overlap (docs don't say they can).

Reply:
Yeah it's a bug:
b/152402742